layout/_layout.swig

```diff
@@ -89,6 +89,7 @@
   {% include '_third-party/needsharebutton.swig' %}
   {% include '_third-party/rating.swig' %}
   {% include '_third-party/mathjax.swig' %}
+  {% include '_third-party/pangu.swig' %}
```

Comment:
> Switch option?

I am not quite clear about that.

The user could turn on/off pangu in `pangu: false` option in `_config.yml`

------

In fact, why I use a third_party swig not just append some if branches in `_scirpts/vendors.swig` is because 

pangu need to invoke `pangu.spacingPage()` to be active.

Is there a more elegant approach?

Reply:
Oh, i mistake. Switch inside:

> {% if theme.pangu %}

Ok, all fine with this.